CHANGELOG.md

```diff
@@ -17,7 +17,7 @@
 ### Documentation
 * Image thumbs generation updated to be Bokeh 2 compatible (#1116)
 * Add new examples for `plot_pair` (#1110)
-
+* Add examples for `psiswl` and `r2_score` (#1129) 
```

Comment:
psislw, the w and l are inverted

Reply:
Sorry for this. It was very careless of me. I'll rectify this.

Reply:
Don't worry, we all make typos :)